app/templates/components/public/ticket-list.hbs

```diff
@@ -131,7 +131,7 @@
                   {{#if this.amountEditable}}
                     <td>{{currency-symbol this.eventCurrency}}</td> <td><Input @value={{this.overridenAmount}} placeholder="Total" /></td>
                   {{else}}
-                    <td>{{currency-symbol this.eventCurrency}} {{format-money this.orderAmount.total}}</td>
+                    <td>{{#if this.orderAmount}}{{currency-symbol this.eventCurrency}}{{/if}} {{format-money this.orderAmount.total}}</td>
```

Comment:
This is not what was mentioned in the issue

Reply:
It says that $ sign should not be shown when the order amount is zero and if talking about country currency, if someone has added paid tickets in the event they also have to choose currency in the event so the currency they will be choosing will be reflected if amount is greater than zero, and i they have chosen only free tickets then no $ sign will be shown

Reply:
there was a bug which I didn't noticed 
Fixed now 
instead of ```this.orderAmount``` I have to use ```this.orderAmount.total```

Reply:
Please let me know if this is not the issue what I am thinking of.